config/rbac/rbac_role.yaml

```diff
@@ -78,18 +78,6 @@ rules:
   - get
   - update
   - patch
-- apiGroups:
```

Comment:
Changing this file is likely moot as it should get clobbered by the autogen in `make manifests`

Reply:
ok that's annoying.